### DISCUSSION TITLE
How to show picture on template card

### DISCUSSION THREAD

**Discussion by DaZeroXB** [piitaya/lovelace-mushroom#1262] [Q&A] [answered]
I am not able to show a pre-uploaded picture on the template card. (https://github.com/piitaya/lovelace-mushroom/blob/main/docs/cards/template.md)

The docs state to just use the `picture` variable.

I did the following:

Uploaded a file via VSCode:
![image](https://github.com/piitaya/lovelace-mushroom/assets/121690461/1c154f74-eff6-48ed-bde2-b9c3495ea637)

path should be: www/tesla_my.png

Tried to use it on template card without success
![image](https://github.com/piitaya/lovelace-mushroom/assets/121690461/cd08b8f7-0120-49c5-98c2-85a5059ae3bd)
![image](https://github.com/piitaya/lovelace-mushroom/assets/121690461/1e271f3c-d687-4c9c-8d03-1e445aa15fc8)

Thanks

**Comment by piitaya** [comment, marked as answer]
Can you try `/local/tesla_my.png` ?

**Reply by DaZeroXB** [reply]
thanks, that worked!
